types/global.d.ts

```diff
@@ -1,5 +1,6 @@
 interface Window { 
   prestashop: any;
+  Theme: any;
```

Comment:
As this is an intern dependency, can't we import the `selectors` object and use something like `Theme: typeof selectors`?

Reply:
We should definitely type it somehow, it will have a lot of benefits

Reply:
`src/js/constants/selectors-map.ts`
```javascript
export interface Selector {
  [key: string]: string | object;
}

export const facetedsearch: Selector = {
  range: '.js-faceted-slider',
  rangeContainer: '.js-faceted-slider-container',
  filterSlider: '.js-faceted-filter-slider',
};
```

`types/global.d.ts`
```javascript
import {Selector} from "@constants/selectors-map";

interface Window { 
  prestashop: any;
  Theme: Selector;
  $: JQueryStatic;
  jQuery: JQueryStatic;
}

```

Reply:
@mparvazi don't use `object`, it's almost like using `any`

and I think this interface would be too generic, we could type it stronger somehow

Reply:
Let's ignore that for now, I don't have the time to take a look rn, I'll create an issue when this gets merged